packages/builders/src/components/selectMenu/StringSelectMenu.ts

```diff
@@ -1,7 +1,8 @@
 import { ComponentType } from 'discord-api-types/v10';
 import type { APIStringSelectComponent, APISelectMenuOption } from 'discord-api-types/v10';
 import { normalizeArray, type RestOrArray } from '../../util/normalizeArray.js';
-import { jsonOptionValidator, optionsLengthValidator, validateRequiredSelectMenuParameters } from '../Assertions.js';
```

Comment:
jsonOptionValidator is not used anywhere anymore, but since we export it it would be breaking to remove it. deprecate maybe?

Reply:
Just did it anyway: b94c5fc03e68a06abf84af224133cba46da7d72c